packages/daemon/src/daemon.js

```diff
@@ -329,7 +329,9 @@ const makeEndoBootstrap = (
         formulaIdentifier,
         formula.worker,
         formula.powers,
-        formula.importPath,
+        formula.specifier ??
+          // @ts-expect-error
+          formula.importPath, // (TODO deprecated)
```

Comment:
This is me trying to be gentle about the need for `endo reset`. All new formulas will use the new format, and the new version can read old formulas. I am not promising forward-compatibility yet, but it was easy to be backward-compatible for this round.